quetz/utils.py

```diff
@@ -24,6 +30,74 @@ def add_entry_for_index(files, subdir, fname, data_bytes):
     )
 
 
+def parse_query(query):
+    accepted_filters = ['channel', 'description', 'summary']
```

Comment:
how about filtering by user (owner)?

Reply:
or uploader. although that is more tricky as that is part of the packageversion data.

Reply:
Yes, we are gonna expand functionality one by one. Right now, all things listed in `PackageSearch` class in `rest_models.py` are considered.

Reply:
Added `uploader` as well